python/ray/serve/_private/config.py

```diff
@@ -117,7 +117,7 @@ class DeploymentConfig(BaseModel):
     )
     max_ongoing_requests: PositiveInt = Field(
         default=DEFAULT_MAX_ONGOING_REQUESTS,
-        update_type=DeploymentOptionUpdateType.NeedsReconfigure,
+        update_type=DeploymentOptionUpdateType.NeedsActorReconfigure,
```

Comment:
What is the behavior if the number of queued requests exceeds the new max? Does the replica simply drain the queue until it reaches the new max before accepting new requests?

Reply:
Yes I believe so

Reply:
yes that's correct